### PR TITLE
chg: attribute list on view event page sort by date issue #1355

### PR DESCRIPTION
**Pull request by cristianbell**
#### What does it do?

`It resolves issue #<1355>`
#### Questions
- [ ] Does it require a DB change?
- [ ] Are you using it in production?
- [ ] Does it require a change in the API (PyMISP for example)?
#### Release Type:
- [ ] Major
- [X] Minor
- [ ] Patch

### CODE REVIEW

**Comment by iglocska** [comment]
There are 3 lines changed - 2 of them are not needed (one is a var_dump left in commented out, the other a comment about an issue - we don't need that, we have git for version control / issue tracking).

For the actual solution, instead of keeping the timestamp in the object twice, it's better to tell the paginator to use the timestamp for the pagination. The actual pagination happens in app/View/Elements/eventattribute.ctp

Line 148 there is this:
<th><?php echo $this->Paginator->sort('date');?></th>

Which can be changed to:

<th><?php echo $this->Paginator->sort('timestamp', 'Date');?></th>

This will tell the paginator to use the timestamp field and still label it Date and then we don't need to store extra junk in memory.

**Comment by cristianbell** [comment]
ok, fixed.

**Comment by iglocska** [comment]
Thanks a lot

File: app/Model/Event.php [hunk 1]
```diff
@@ -2629,7 +2629,7 @@ public function rearrangeEventForView(&$event, $passedArgs = array(), $all = fal
 		$correlatedAttributes = isset($event['RelatedAttribute']) ? array_keys($event['RelatedAttribute']) : array();
 		$correlatedShadowAttributes = isset($event['RelatedShadowAttribute']) ? array_keys($event['RelatedShadowAttribute']) : array();
 		foreach ($event['Attribute'] as $attribute) {
-			if ($filterType && !in_array($filterType, array('proposal', 'correlation', 'warning'))) if (!in_array($attribute['type'], $this->Attribute->typeGroupings[$filterType])) continue;
+    			if ($filterType && !in_array($filterType, array('proposal', 'correlation', 'warning'))) if (!in_array($attribute['type'], $this->Attribute->typeGroupings[$filterType])) continue;
```
Comment (rotanid) [inline app/Model/Event.php:2632]: your IDE added some spaces here :D

**Comment by rotanid** [comment]
in the future, please don't add commit after commit in a PR but do "commit --amend" oder "rebase -i" etc. instead to not pollute MIPSs commitlog with unnecessary commits, makes reading the history easier (and space and performance etc)

**Pull request merged**